SpecializedServers/SVO/Games/PS2/GT4Beta.cs

```diff
@@ -0,0 +1,78 @@
+                                            "<DATA dataType=\"DATA\" name=\"playerStatsURI\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/account/stats\" />\r\n\t\t" +
+                                            "<DATA dataType=\"DATA\" name=\"playerProfileURI\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/account/profile\" />\r\n\t\t" +
+                                            "<DATA dataType=\"URI\" name=\"LogoutURL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/account/Account_Logout.jsp\"/>\r\n\t\t\r\n\t\t" +
+                                            "<DATA dataType=\"URI\" name=\"gameTimeTrialCreateURL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/game/Game_TimeTrial_Create_Submit.jsp\"/>\r\n\t\t" +
+                                            "<DATA dataType=\"URI\" name=\"gameArcadeCreateURL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/game/Game_Arcade_Create_Submit.jsp\"/>\r\n\t\t" +
+                                            "<DATA dataType=\"URI\" name=\"gameFinishURL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/game/Game_Finish_Submit.jsp\"/>\r\n\t\t" +
+                                            "<DATA dataType=\"DATA\" name=\"loginE3URL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/account/loginE3.jsp\" />\r\n\t\t" +
+                                            "<DATA dataType=\"DATA\" name=\"GetTrialRankingUrl\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/ranking/TrialRanking.jsp\" />\r\n\t\t" +
+                                            "<DATA dataType=\"DATA\" name=\"statsLocationURL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/stats/statsLocation.jsp\" />\r\n\t\t" +
+                                            "<DATA dataType=\"DATA\" name=\"statsTrialURL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/stats/statsTrial.jsp\" />\r\n\t\t" +
+                                            "<DATA dataType=\"DATA\" name=\"statsTrialPositionURL\" value=\"http://gt4-pubeta.svo.pdonline.scea.com:10060/GT4_XML/stats/statsTrialPosition.jsp\" />\r\n\t\t" +
+                                            "<BROWSER_INIT name=\"init\" />\r\n\t" +
+                                            "</URL_List>\r\n" +
+                                            "</XML>");
+
+                                        resp.ContentLength64 = uriStore.Length;
+                                        resp.OutputStream.Write(uriStore);
+#if DEBUG
+                                        LoggerAccessor.LogInfo($"Start URIStore for GT4 Online Public Beta PS2 SENT!");
+#endif
+                                        break;
+                                }
+
+                                break;
+                            }
+                    }
+                }
+                catch (Exception e)
+                {
+
```

Comment:
You need to add the error log and the response status : 

LoggerAccessor.LogError($"[SVO] - GT4Beta_SVO thrown an assertion - {ex}");
response.StatusCode = (int)System.Net.HttpStatusCode.InternalServerError;